src/Mod/AddonManager/Widgets/addonmanager_widget_view_selector.py

```diff
@@ -114,7 +114,7 @@ def _setup_ui(self):
         self.composite_button = QtWidgets.QToolButton(self)
         self.composite_button.setObjectName("composite_button")
         qtcore_major_ver = int(float(QtCore.qVersion()[0]))
-        qtcore_minor_ver = int(float(QtCore.qVersion()[2]))
+        qtcore_minor_ver = int(float(QtCore.qVersion()[2] + QtCore.qVersion()[3]))
```

Comment:
I'm not sure what this code is really doing, but I don't think it's what you want. qVersion is a string with three components in it, and isn't really the best way of checking this. I think you are better off using `QLibraryInfo.version()`, from which you can directly access `majorVersion`, `minorVersion`, and `microVersion`.